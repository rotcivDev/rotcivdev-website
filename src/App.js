import React from 'react';
import { Switch, Route } from 'react-router-dom';

import HomePage from './pages/homepage';
import Profile from './pages/profile';

function App() {
  return (
    <Switch>
      <Route exact path='/' component={HomePage}/>
      <Route exact path='/profile' component={Profile}/>
    </Switch>
  );
}

export default App;
