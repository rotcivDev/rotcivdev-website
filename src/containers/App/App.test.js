import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App Component', () => {
  test('Should render App in homepage', () => {
    expect(render(<App />)).toMatchSnapshot();  
  
  });

})
