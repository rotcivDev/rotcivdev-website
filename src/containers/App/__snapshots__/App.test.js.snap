// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Should render App in homepage 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homepage u-center-text"
        data-testid="homepageContainer"
      >
        <header
          class="homepage__title--container"
          data-testid="header"
        >
          <h1
            class="title"
          >
            rotcivDev's Coding Adventures
          </h1>
        </header>
        <main
          data-testid="main"
        >
          <div
            class="homepage__slide-show"
          />
        </main>
        <section
          class="homepage__enter-button-container"
        >
          <a
            class="homepage__enter-button"
            data-testid="enterbutton"
            href="/profile"
          >
            Enter
          </a>
        </section>
        <footer
          class="homepage__footer"
          data-testid="footer"
        >
          <a
            class="homepage__footer--link u-underline-onhover"
            href="https://www.github.com/rotcivdev"
          >
            github: rotcivDev
          </a>
          <h5
            class="homepage__footer--message"
          >
            Development with
            <span
              aria-label="heart emoji"
              class="homepage__footer--message-heart"
              role="img"
            >
              ♥️
            </span>
            to improve life!
          </h5>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homepage u-center-text"
      data-testid="homepageContainer"
    >
      <header
        class="homepage__title--container"
        data-testid="header"
      >
        <h1
          class="title"
        >
          rotcivDev's Coding Adventures
        </h1>
      </header>
      <main
        data-testid="main"
      >
        <div
          class="homepage__slide-show"
        />
      </main>
      <section
        class="homepage__enter-button-container"
      >
        <a
          class="homepage__enter-button"
          data-testid="enterbutton"
          href="/profile"
        >
          Enter
        </a>
      </section>
      <footer
        class="homepage__footer"
        data-testid="footer"
      >
        <a
          class="homepage__footer--link u-underline-onhover"
          href="https://www.github.com/rotcivdev"
        >
          github: rotcivDev
        </a>
        <h5
          class="homepage__footer--message"
        >
          Development with
          <span
            aria-label="heart emoji"
            class="homepage__footer--message-heart"
            role="img"
          >
            ♥️
          </span>
          to improve life!
        </h5>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
